core/src/test/java/fr/sncf/osrd/train/StopTests.java

```diff
@@ -176,6 +182,7 @@ public void testCorrectNumberGeneratedEvents() {
         assertEquals(6, nStops);
     }
 
+    @Disabled
```

Comment:
Why did you disabled this test?

Reply:
It was a test using LinearMargin, so I just disabled it